src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -210,17 +210,19 @@ const useCloudEnvironmentPolling = (googleProject, workspace) => {
   const saturnWorkspaceNamespace = workspace?.workspace.namespace
   const saturnWorkspaceName = workspace?.workspace.name
 
+  console.log(workspace)
+
   const reschedule = ms => {
     clearTimeout(timeout.current)
     timeout.current = setTimeout(refreshRuntimesSilently, ms)
   }
   const load = async maybeStale => {
     try {
+      const runtimeFilters = !!googleProject ? { googleProject } : {saturnWorkspaceNamespace, saturnWorkspaceName}
```

Comment:
I'd like to use lodash to filter all undefined values if possible. Imagine a case where for some runtime this label isn't defined, passing `x: undefined` as a filter to leo could cause the filter it to match no runtimes, as opposed to runtimes without those labels present.

This code should work, tested in the console
`const runtimeFilters = _.pickBy(x => x !== undefined, { googleProject, saturnWorkspaceName, saturnWorkspace, creator: getUser().email }
const [newDisks, newRuntimes] = !!workspace ? await Promise.all([
        Ajax(signal).Disks.list({ googleProject, creator: getUser().email, includeLabels: 'saturnApplication,saturnWorkspaceName', saturnWorkspaceName }),
        Ajax(signal).Runtimes.listV2(runtimeFilters)
`

Reply:
The one edge case I'm concerned about is if there are multiple runtimes (pre-PPW) that share a `googleProject` and don't have `saturnWorkspaceName` and `saturnWorkspaceNamespace` labels

Reply:
there're about 296 google projects that have more than 1 runtime, and the runtimes don't have `saturnWorkspaceName`

Query
```
select count(*), cloudContext from CLUSTER LEFT JOIN `LABEL` as L on CLUSTER.id = L.`resourceId` where CLUSTER.runtimeName LIKE 'saturn%' and CLUSTER.status ="Stopped" and L.`resourceType`="runtime" and NOT EXISTS (select * from LABEL AS L2 where L2.key="saturnWorkspaceName" and L2.`resourceId`=L.resourceId and L2.`resourceType`="runtime") group by CLUSTER.cloudContext having count(*)>1 ;
```

Reply:
We talked about this offline and decided that 
```
const cloudEnvFilters = _.pickBy(l => !_.isUndefined(l),
        { googleProject, saturnWorkspaceName, saturnWorkspaceNamespace, creator: getUser().email }
      )
```
will account for both pre PPW workspaces and post PPW. This logic will make it so that whatever is defined in `{ googleProject, saturnWorkspaceName, saturnWorkspaceNamespace, creator: getUser().email }` will be used to filter runtimes and disks